dictionaries/software-terms/src/software-terms.txt

```diff
@@ -1704,3 +1704,4 @@ zip
 zlib
 ZPool
 ZPools
+Databricks
```

Comment:
Databricks is a company and a tool, but not a software term. Please add it to software tools.

Reply:
done